dcos/dcos-ui
# DCOS-50974: Replace Secrets select with Typeahead

## Testing
Before testing, make sure there are a few secrets added

In the service or job form:
1. Go to the "Secrets" tab
2. Click "+ Add a Secret"
3. Focus the Secret field

Selecting an item from the Typeahead dropdown should set the value of the input and update the JSON.
The list in the typeahead dropdown should filter as you type 

## Trade-offs



## Dependencies
Same branch name in dcos-ui-plugins-private. PR: https://github.com/mesosphere/dcos-ui-plugins-private/pull/1010

## Screenshots
![TypeaheadDemo_jobs](https://user-images.githubusercontent.com/2313998/57934260-937c6300-78bf-11e9-979a-259a7fb6da67.gif)
